src/index.ts

```diff
@@ -54,6 +57,11 @@ export function viteSvgToWebfont<T extends GeneratedFontTypes = GeneratedFontTyp
     return {
         name: 'vite-svg-2-webfont',
         enforce: 'pre',
+        api: {
```

Comment:
Hi, I was unable to find any docs about this, could you share some docs about how this works and where it can be used?
Also, care to document this in the `README`?

Reply:
Hello! Here's the docs: https://rollupjs.org/plugin-development/#direct-plugin-communication

I'll update the readme a little bit later.

Reply:
Thanks for the reference!
I'd be happy to merge once there is some `readme` about this

Reply:
Thank you! I'll plan to update the readme tomorrow. Sorry for the delay :(